src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -0,0 +1,526 @@
+//! This sink sends data to Google Chronicles unstructured log entries endpoint.
```

Comment:
Just noticing this is the primary inline documentation for the file (in reference to the "REVIEWING.md" guide).
Though I'm glancing at a few other sink implementations and, not seeing a whole lot of in line documentation in those either 🤷